### PR TITLE
t-tabularToRelational: Changes in user experience. Improved stability.

### PR DESCRIPTION
**Pull request by eea02**
Improvements of user expericence. Refactored key parts of DPU to increase stability. 
Implemented part of SQL data type guessing algorithm. This feature was later abandoned (code remains, but is not used).

### CODE REVIEW

File: t-tabularToRelational/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	<artifactId>uv-t-tabularToRelational</artifactId>
 	<name>T-TabularToRelational</name>
 	<description>Parse tabular file to relational data unit.</description>
-	<version>2.0.1</version>
+	<version>2.0.2</version>
```
Comment (skrchnavy) [inline t-tabularToRelational/pom.xml]: Shall be snapshot version, and it is not only bugfix but also change of configuration => 2.1.0. (also Readme.md is affected)

File: t-tabularToRelational/src/main/java/eu/unifiedviews/plugins/transformer/tabulartorelational/TabularToRelational.java [hunk 2]
```diff
@@ -42,9 +43,12 @@
     @ExtensionInitializer.Init
     public FaultTolerance faultTolerance;
 
-    @ExtensionInitializer.Init(param = "eu.unifiedviews.plugins.transformer.tabulartorelational.TabularToRelationalConfig__V1")
+    @ExtensionInitializer.Init(param = "eu.unifiedviews.plugins.transformer.tabulartorelational.TabularToRelationalConfig__V2")
```
Comment (skrchnavy) [inline t-tabularToRelational/src/main/java/eu/unifiedviews/plugins/transformer/tabulartorelational/TabularToRelational.java]: Isn't ```TabularToRelationalConfig__V2.java``` missing in the commit?

File: t-tabularToRelational/pom.xml (hunk 1 above)
Comment (eea02) [inline t-tabularToRelational/pom.xml]: Version fixed.

File: t-tabularToRelational/src/main/java/eu/unifiedviews/plugins/transformer/tabulartorelational/TabularToRelational.java (hunk 2 above)
Comment (eea02) [inline t-tabularToRelational/src/main/java/eu/unifiedviews/plugins/transformer/tabulartorelational/TabularToRelational.java]: Typo fixed. There was no change in configuration class.

**Comment by skrchnavy** [comment]
fix also #192, if donw, please provide commit where it is fixed in #192

**Comment by tomas-knap** [comment]
Before merging, please fix the issue #192, as required by @skrchnavy

**Comment by tomas-knap** [comment]
Approved, but please address the comment from @skrchnavy above

**Pull request merged**